test/lib/main.spec.js

```diff
@@ -191,17 +193,18 @@ describe('main', () => {
     expect(actual).to.be.undefined;
   });
 
-  it('should provide an error if action requires an instance and apiUrl does not respond', async() => {
-    mocks.request.get.rejects();
-    await main([...normalArgv, 'upload-app-forms']);
-    expect(mocks.error.callCount).to.eq(1);
-    expect(mocks.error.args[0][0]).to
-      .eq('Failed to get a response from http://api. Maybe you entered the wrong URL, wrong port or the instance is not started? Please check and try again.');
+  it('should return earlier with false value if api is not available', async () => {
+    apiAvailable = sinon.stub().resolves(false);
```

Comment:
I think the re-assignment is not required here. The variable is already a stub. 
```suggestion
    apiAvailable.resolves(false);
```